stack/adding-a-new-package.rst

```diff
@@ -119,9 +119,8 @@ For ``afw`` this is located at https://github.com/lsst/afw/settings/branches/ an
 In the "Branch protection rules" section of that page you will have to click on "Add rule" to create a rule for ``main``.
 First, add in ``main`` as the branch name pattern.
 Second, enable ``Require a pull request before merging``, but disable ``Require approvals``.
-Third, enable status checks, require that branches be up to date before merging, and add the ``lint`` GitHub action to the list of required status checks.
-To enable the ``lint`` GitHub action, type ``lint`` into the search box and select the ``lint`` GitHub action.
-Finally, include Administrators in these protections, since it's all too easy to make a mistake without realizing you have special override powers.
```

Comment:
This option no longer exists. I believe the equivalent is to leave the "Do not allow bypassing the above settings" unchecked, which is the default.